spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/TracingProperties.java

```diff
@@ -53,4 +74,119 @@ public void setProbability(float probability) {
 
 	}
 
+	public static class Baggage {
+
+		/**
+		 * Enables correlating the baggage context with logging contexts.
```

Comment:
For consistency with other enabled properties, this should be something like "Whether to enable correlation of the baggage context with logging contexts"